ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -10,7 +10,7 @@ spec:
   resolution: "Go to System Settings/Software Update and install the latest updates manually"
   query: SELECT 1 FROM os_version WHERE version >= '13.1';
   purpose: Informational
-  tags: compliance, CIS, CIS_Level1, CIS1.1
+  tags: compliance, CIS, CIS_Level1, CIS-macos13-ventura-1.1
```

Comment:
macOS 13 is called Ventura, so maybe a bit redundant to use both? We could use `macos-13` or `macos-ventura`.